e2etest/newe2e_resource_managers_blob.go

```diff
@@ -83,8 +83,14 @@ func (b *BlobServiceResourceManager) ListContainers(a Asserter) []string {
 	return out
 }
 
-func (b *BlobServiceResourceManager) URI() string {
-	return blobStripSAS(b.internalClient.URL())
+func (b *BlobServiceResourceManager) URI(a Asserter, withSas bool) string {
+	base := blobStripSAS(b.internalClient.URL())
+
+	if withSas {
```

Comment:
why do we need to apply sas below? does the internalClient.URL not contain SAS?

Reply:
Nope. It's using account key based validation.